docs/concepts/schemas.md

```diff
@@ -141,6 +136,27 @@ time the statement is first executed.
 
 #### With a key column
 
+The following statement shows how to create a new `pageviews` stream by reading
+from a {{ site.ak }} topic that has Avro-formatted key and message values.
+
+```sql
+CREATE STREAM pageviews WITH (
+    KAFKA_TOPIC='pageviews-avro-topic',
+    KEY_FORMAT='AVRO',
+    VALUE_FORMAT='AVRO'

```

Comment:
should we also mention `FORMAT='AVRO'`? I suspect this might be somewhat common

Reply:
Were you imagining a tip saying that this is equivalent to simply specifying `FORMAT`, updating these examples to use `FORMAT`, or something else? The `FORMAT` property is documented in https://github.com/confluentinc/ksql/blob/master/docs/developer-guide/ksqldb-reference/create-stream.md (and similar for CT, CSAS, and CTAS) which feels like the more appropriate place to formally introduce it, but I can certainly slip in a mention of it here as well.

Reply:
Discussed offline -- leaving this as is for now. Will see if we can consolidate information about keys into a natural place in a subsequent PR.